README.md

```diff
@@ -79,8 +79,8 @@ fn main() {
   // response is well formed.)
   for header in response.headers.iter() {
     println!("{}: {}",
-        str::from_utf8(&header.0).unwrap(),
```

Comment:
This should actually be `str::from_utf8(header.name())`. The `name` field is private on the `Header` struct and exposed via the correspondingly named method.

See the [example](https://github.com/mlalic/solicit/blob/73a283602342cc7aa281fd1ba4430ecb67cebb52/src/client/simple.rs#L72-L94) in the docs of the client.